docs/core/resilience/http-resilience.md

```diff
@@ -243,55 +162,38 @@ The preceding code:
 - Adds a circuit breaker strategy with a sampling duration of 10 seconds, a failure ratio of 0.2 (20%), a minimum throughput of three, and a predicate that handles `RequestTimeout` and `TooManyRequests` HTTP status codes to the resilience builder.
 - Adds a timeout strategy with a timeout of five seconds to the resilience builder.
 
-There are many options available for each of the resilience strategies. For more information, see the [Polly docs: Strategies](https://www.pollydocs.org/strategies).
+There are many options available for each of the resilience strategies. For more information, see the [Polly docs: Strategies](https://www.pollydocs.org/strategies). For more information about configuring `ShouldHandle` delegates, see [Polly docs: Fault handling in reactive strategies](https://www.pollydocs.org/strategies/index.html#fault-handling).
```

Comment:
The links to pollydocs.org seem to mix and match "page.html" links and not. I think you can update them all to be like the `https://www.pollydocs.org/strategies` one.